docgen.mjs

```diff
@@ -5,7 +5,16 @@ import _ from 'lodash-es';
 import path from 'path';
 import parseSCSS from './docgen-scss.mjs';
 
-const componentFiles = glob.sync("./src/components/*/*.vue");
+const componentFiles = glob.sync(
+  "./src/components/**/*.vue",
+  { 
+    ignore: [
+      './src/components/**/examples/**/*',
+      './src/components/**/components/*',
+      './src/components/icon/comps/*.vue'
+    ]
+  }
+);
```

Comment:
Allows the glob to find more deeply nested vue components (e.g. the fluid presets), adds ignore rules to avoid globbing icons, examples, helpers.